docker-compose.yml

```
@@ -0,0 +1,27 @@
+    depends_on:
+      - prometheus
+    restart: always
+    networks:
+      - monitoring
+    environment:
+      - GF_SECURITY_ADMIN_USER=admin
+      - GF_SECURITY_ADMIN_PASSWORD=admin
+
+networks:
+  monitoring:
+    driver: bridge
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)